codec/legacy/codec.go

```diff
@@ -15,5 +16,16 @@ func init() {
 	Cdc = codec.NewLegacyAmino()
 	cryptocodec.RegisterCrypto(Cdc)
 	codec.RegisterEvidences(Cdc)
-	Cdc.Seal()
```

Comment:
I'm not so comfortable just removing this Seal like this. is this because of https://github.com/cosmos/cosmos-sdk/pull/7987/files#r529554643 ?

Reply:
not quite. Because if we only use the `codec/legacy.Cdc`, it is inevitable to register other structures or interfaces, and it is not suitable to call `seal()` here.

Reply:
Ah, that makes sense.